libs/types/document.ts

```diff
@@ -1,5 +1,8 @@
 import { type Locale } from "./core.js";
-import type { SupportStatus } from "./web-features.js";
+
+// web-features doesn't export this type directly so we need to do a little typescript magic:
+import type { features } from "web-features";
+type SupportStatus = (typeof features)[keyof typeof features]["status"];
```

Comment:
Since we define this here, could we just export this as `WebFeatureSupportStatus` and import above, to avoid duplication?

Reply:
Again, I don't want to diverge from the names/module import hierarchy compared to importing from web-features directly, which we can hopefully do soon

Reply:
Ok